MapboxVision/VisionManager/ManagerDependencies.swift

```diff
@@ -78,7 +81,10 @@ struct ReplayDependencies {
     let player: VideoPlayable
 
     static func `default`(recordPath: String) throws -> ReplayDependencies {
-        guard let reachability = Reachability() else {
+        let reachability: Reachability
+        do {
+            reachability = try Reachability()
+        } catch {
```

Comment:
I'd prefer `guard let reachability = try? Reachability() else {` but it's ok too